index.d.ts

```diff
@@ -2234,7 +2236,7 @@ declare namespace Eris {
     createCommand(command: ApplicationCommandStructure): Promise<ApplicationCommand>;
     createGroupChannel(userIDs: string[]): Promise<GroupChannel>;
     createGuild(name: string, options?: CreateGuildOptions): Promise<Guild>;
-    createGuildCommand(guildID: string, command: ApplicationCommandStructure): Promise<ApplicationCommand>;
+    createGuildCommand(guildID: string, command: Omit<ApplicationCommandStructure, 'DMPermission'>): Promise<Omit<ApplicationCommand, 'DMPermission'>>;
```

Comment:
```suggestion
    createGuildCommand(guildID: string, command: Omit<ApplicationCommandStructure, 'dmPermission'>): Promise<Omit<ApplicationCommand, 'dmPermission'>>;
```
keys should be camel cased